resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/DataprocWorkerChecker.scala

```diff
@@ -40,7 +40,10 @@ object DataprocWorkerChecker {
             .getCluster(runtime.r.googleProject, regionName, DataprocClusterName(runtime.r.runtimeName))
           runtime <- clusterOpt.flatTraverse { c =>
             val doesPrimaryWorkerMatch =
-              runtime.workerConfig.numberOfWorkers.getOrElse(0) == c.getConfig.getWorkerConfig.getNumInstances
+              if (c.getConfig.getWorkerConfig.getNumInstances == 0)
+                true //pre-emptible workers can disappear, which isn't an anomaly if it doesn't match what we have in Leo DB
```

Comment:
It'd be good to modify the comment a bit to the effect of:

```scala
// We remove preemptibles before stopping clusters. Therefore, we don't consider this case an anomaly.
```

Reply:
I see..I didn't know we remove preemptiles on our end